synapse/models/orgs.py

```diff
@@ -87,56 +141,21 @@ def getBaseModels():
                 ]),
 
                 ('ou:owns', {'ptype': 'sepr', 'sep': '/', 'fields': 'owner,ou:org|owned,ou:org'}, [
-                ]),
-
-                ('ou:hasalias', {'ptype': 'ou:hasalias'}, (
```

Comment:
@therealsilence @invisig0th - Migrating this form will not work because `ou:alias` is not a form. Should I make `ou:alias` a form, should I leave these forms be, or should I delete them?

Reply:
I dont think these exist right now, so go ahead and leave them out of the migration

Reply:
OK - I will not remove them from the model and will not remove the tests and not migrate them.